An Untitled Mining Game/An Untitled Mining Game/objects/obj_player/Step_0.gml

```diff
@@ -10,21 +10,21 @@ y = clamp(y, 0, room_height-16);
 
 
 //Keep player from moving into solid objects
-if (!(place_free(x - 2, y)) and (hspeed < 0)) hspeed = 0;
-if (!(place_free(x + 2, y)) and (hspeed > 0)) hspeed = 0;
+if (!(place_free(x - hspeed, y)) and (hspeed < 0)) hspeed = 0; // x+ hspeed
+if (!(place_free(x + hspeed, y)) and (hspeed > 0)) hspeed = 0;
 
-if (!(place_free(x , y - 2)) and (vspeed < 0)) vspeed = 0;
-if (!(place_free(x, y + 2)) and (vspeed > 0)) vspeed = 0;
+if (!(place_free(x , y - vspeed)) and (vspeed < 0)) vspeed = 0;
+if (!(place_free(x, y + vspeed)) and (vspeed > 0)) vspeed = 0;
```

Comment:
I remember this from our pods in-class discussion, but this change was not mentioned in the PR description!